### DISCUSSION TITLE
Support of grouped operations for lags and rolling windows?

### DISCUSSION THREAD

**Discussion by flostracke** [feature-engine/feature_engine#485] [Q&A]
Hi, 

i just discovered yorur package and it looks very useful! Great work! I am also looking forward to your course feature engineering for timeseries :)

Now to my questions: Is the creations for lag features possible per group? I couldn't find anything in the documentation, therefore I would assume this is currently not possible? 

Thank you very much and best regards,
Florian

**Comment by solegalli** [comment]
Hi Florian,

By groups you mean, say you have an extra variable with Country, and then you want to create the lags for each country?

That is not implemented yet. We have it on the radar. But you know, we are short of human power, so it might take a while :)
